src/Microsoft.DotNet.Compiler.Common/AssemblyInfoFileGenerator.cs

```diff
@@ -68,9 +69,15 @@ public static string GenerateFSharp(AssemblyInfoOptions metadata)
                 [typeof(AssemblyVersionAttribute)] = EscapeCharacters(metadata.AssemblyVersion?.ToString()),
                 [typeof(AssemblyInformationalVersionAttribute)] = EscapeCharacters(metadata.InformationalVersion),
                 [typeof(AssemblyCultureAttribute)] = EscapeCharacters(metadata.Culture),
-                [typeof(NeutralResourcesLanguageAttribute)] = EscapeCharacters(metadata.NeutralLanguage),
-                [typeof(TargetFrameworkAttribute)] = EscapeCharacters(metadata.TargetFramework)
+                [typeof(NeutralResourcesLanguageAttribute)] = EscapeCharacters(metadata.NeutralLanguage)
             };
+            
+            NuGetFramework targetFramework = string.IsNullOrEmpty(metadata.TargetFramework) ? null : NuGetFramework.Parse(metadata.TargetFramework);
```

Comment:
The unit test "TestProjectWithCultureSpecificResource" failed, so I added the null check. Is this unintended?
```
Project TestProjectWithCultureSpecificResource (DNXCore,Version=v5.0) will be compiled because some of its inputs were newer than its oldest output.
Compiling TestProjectWithCultureSpecificResource for DNXCore,Version=v5.0
System.ArgumentNullException: Value cannot be null.
Parameter name: folderName
   at NuGet.Frameworks.NuGetFramework.Parse(String folderName, IFrameworkNameProvider mappings)
   at Microsoft.DotNet.Cli.Compiler.Common.AssemblyInfoFileGenerator.IsAllowV4Attributes(AssemblyInfoOptions metadata) in C:\dev\dotnet-cli\src\Microsoft.DotNet.Compiler.Common\AssemblyInfoFileGenerator.cs:line 91
   at Microsoft.DotNet.Cli.Compiler.Common.AssemblyInfoFileGenerator.GetProjectAttributes(AssemblyInfoOptions metadata) in C:\dev\dotnet-cli\src\Microsoft.DotNet.Compiler.Common\AssemblyInfoFileGenerator.cs:line 75
   at Microsoft.DotNet.Cli.Compiler.Common.AssemblyInfoFileGenerator.Generate(AssemblyInfoOptions metadata, IEnumerable`1 sourceFiles) in C:\dev\dotnet-cli\src\Microsoft.DotNet.Compiler.Common\AssemblyInfoFileGenerator.cs:line 21
   at Microsoft.DotNet.Tools.Resgen.ResourceAssemblyGenerator.Generate(ResourceSource[] sourceFiles, Stream outputStream, AssemblyInfoOptions metadata, String assemblyName, String[] references) in C:\dev\dotnet-cli\src\dotnet\commands\dotnet-resgen\ResourceAssemblyGenerator.cs:line 59
   at Microsoft.DotNet.Tools.Resgen.ResgenCommand.Execute() in C:\dev\dotnet-cli\src\dotnet\commands\dotnet-resgen\ResgenCommand.cs:line 36
   at Microsoft.DotNet.Tools.Resgen.ResgenCommand.Run(String[] args) in C:\dev\dotnet-cli\src\dotnet\commands\dotnet-resgen\Program.cs:line 63
```